frontend/src/app/modules/common/collapsible-section/collapsible-section.component.ts

```diff
@@ -28,7 +28,6 @@
 
 
 import {Component, ElementRef, OnInit, ViewChild} from "@angular/core";
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 
 export const collapsibleSectionAugmentSelector = 'collapsible-section-augment';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/modules/common/collapsible-section/collapsible-section.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>